troubleshooting/out-of-memory.md

```diff
@@ -35,13 +35,13 @@ sudo yum clean all
 qvm-remove <VMname>
 ~~~
 
-With this method you lose one VM data, but it'll more securely work.
+With this method you lose the data of one VM, but it'll more reliably work.
```

Comment:
"more ___ work" -> "___ more reliably".

also this whole page is kinda misleading. this is about being out of disk space, which is a separate issue from being out of memory...

are we telling people to delete their data when we should really be telling them to buy more ram or reduce per-vm ram allocation?